src/puppetbridge.ts

```diff
@@ -878,7 +934,13 @@ export class PuppetBridge extends EventEmitter {
 		}
 		const puppetMxid = await this.provisioner.getMxid(room.puppetId);
 		if (event.sender !== puppetMxid) {
-			return; // this isn't our puppeted user, so let's not do anything
+			if (!this.config.relay.enabled) {
+				return; // relaying not enabled, don't allow message from other user
+			} else if (!this.provisioner.canCreate(event.sender)) {
```

Comment:
Shouldn't this maybe be a separate permission? As, if you whitelist many people, many will also be able to create puppet channels then, causing your server to potentially suddenly have tons of slack connections open for all the people maliciously making bridges?

Reply:
Maybe its own blacklist/whitelist for this setting?